server/apiv2/router.go

```diff
@@ -34,7 +38,12 @@ const apiV2Prefix = "/pd/api/v2/"
 
 // NewV2Handler creates a HTTP handler for API.
 func NewV2Handler(_ context.Context, svr *server.Server) (http.Handler, server.ServiceGroup, error) {
-	gin.SetMode(gin.ReleaseMode)
+	once.Do(func() {
+		// These global modification will be effective only for the first invoke.
+		_ = godotenv.Load()
```

Comment:
What's this for?

Reply:
Copied from the dashboard, I guess because it supports configuring the mode through environment variables.